src/common/base64/avx2/base64_intel_avx2.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020, Alfred Klomp <git@alfredklomp.com>
```

Comment:
also, you might want to document the source of this library in the commit message.

Reply:
OK.
I'll add the source code url path in the commit message.

Reply:
Done